app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            "{"
+            + f"Name: {self.name}, "
+            + f"Health: {self.health}, "
+            + f"Hidden: {self.hidden}"
+            + "}"
```

Comment:
Is really `+` needed here?

Reply:
Otherwise, flake8 complains about the long string.